source/fluid/image_view.d

```diff
@@ -82,6 +85,15 @@ class ImageView : Node {
 
         }
 
+        Image texture(Image image) @system {
```

Comment:
I think introducing a texture overload that takes an image is fine, but why make `ImageView` a template?.

Reply:
I did this so that it can be used with any image format that one wants, so long as it has a `draw` function.